rs/nns/governance/src/governance.rs

```diff
@@ -135,7 +135,7 @@ pub mod tla;
 #[cfg(feature = "tla")]
 pub use tla::{
     claim_neuron_desc, split_neuron_desc, tla_update_method, InstrumentationState, ToTla,
-    TLA_INSTRUMENTATION_STATE, TLA_TRACES,
```

Comment:
I don't see where these are used, but I assume it is from macro expansion?

Reply:
Yep, the macros assume these variables to be in scope (I added some doc comments to the macros)